foundry.toml

```diff
@@ -12,11 +12,14 @@ remappings = [
     'ds-test/=lib/forge-std/lib/ds-test/src/',
     'erc4626-tests/=lib/openzeppelin-contracts/lib/erc4626-tests/',
     'forge-std/=lib/forge-std/src/',
-    '@openzeppelin/=lib/openzeppelin-contracts/',
     '@openzeppelin/contracts/=lib/openzeppelin-contracts/contracts/',
     'openzeppelin-contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/',
     'openzeppelin-contracts/=lib/openzeppelin-contracts/',
+    '@openzeppelin/contracts-upgradeable/=lib/openzeppelin-contracts-upgradeable/contracts/',
+    'openzeppelin-foundry-upgrades/=lib/openzeppelin-foundry-upgrades/src/',
+    'solidity-stringutils/=lib/openzeppelin-foundry-upgrades/lib/solidity-stringutils/',
```

Comment:
May you just rerun `forge remappings` and remove what's not necessary to be present here?